### PR TITLE
updated 1-execute-query-mutation to using useReducer hook

### PR DESCRIPTION
**Pull request by idkjs**
Per https://github.com/FormidableLabs/reason-urql/pull/71#issuecomment-504480956

### CODE REVIEW

File: examples/1-execute-query-mutation/src/Previewer.re [hunk 1]
```diff
@@ -101,117 +101,108 @@ type state = {
...
+          ->str
+        </div>
+        <button
+          className={Styles.colors.query->Styles.button}
+          onClick={_event => executeQuery()}>
+          "Execute Query"->str
+        </button>
+      </section>
+      {switch (String.length(state.query)) {
+       | 0 => ReasonReact.null
```
Comment (parkerziegler) [inline examples/1-execute-query-mutation/src/Previewer.re]: While we're here would you mind switching this to `React.null`? Just want to get rid of all `ReasonReact` references while this is fresh ❤️

Comment (idkjs) [reply examples/1-execute-query-mutation/src/Previewer.re]: Also changed a few `ReasonReact.string` to `Reason.string`. Unless you didn't want that changed?

**Comment by idkjs** [comment]
Done.

**Comment by parkerziegler** [comment]
@all-contributors add @idkjs for code, example.

**Pull request merged**